spec/forms/teacher_interface/alternative_name_form_spec.rb

```diff
@@ -1,28 +1,78 @@
 require "rails_helper"
 
 RSpec.describe TeacherInterface::AlternativeNameForm, type: :model do
-  describe "validations" do
-    it { is_expected.to validate_presence_of(:application_form) }
-  end
-
   let(:application_form) { build(:application_form) }
-
-  describe "#valid?" do
-    subject(:valid?) { form.valid? }
-
-    let(:form) do
+  let(:has_alternative_name){}
+  let(:alternative_family_name){}
+  let(:alternative_given_names){}
```

Comment:
```suggestion
  let(:has_alternative_name) { nil }
  let(:alternative_family_name) { nil }
  let(:alternative_given_names) { nil }
```

Better to be explicit here I think so it's clear, also I reckon it makes more sense to define these within the `describe "validations"` block as it's closer to where they're used (they're not used in the `describe "#save"` block so they don't need to be accessible there for example).